### PR TITLE
Feat/release notes

### PR DESCRIPTION
**Pull request by Mohith93**
Created Release Documentation for Multiwoven tracking monthly.

### CODE REVIEW

File: release-notes/July_2024.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- **Role Descriptions**: Updated role descriptions for clearer understanding and easier management.
+- **JWT Secret Configuration**: JWT secret is now configurable from environment variables, boosting security practices.
+- **MariaDB README Update**: Updated the README file to include the latest information on MariaDB connectors.
+- **Logout Authorization**: Streamlined the logout process by skipping unnecessary authorization checks.
+- **Sync Record JSON Error**: Added a JSON error field in sync records to enhance error tracking and debugging.
+- **MariaDB DockerFile Update**: Added `mariadb-dev` to the DockerFile to better support MariaDB integrations.
+- **Signup Error Response**: Improved the clarity and detail of signup error responses.
+- **Role Policies Update**: Refined role policies for enhanced access control and security.
+- **Pundit Policy Enhancements**: Applied Pundit policies at the role permission level, ensuring robust authorization management.
+
```
Comment (sumitd94) [inline release-notes/July_2024.mdx:83]: Can we remove this extra white spaces?

File: release-notes/August_2024.mdx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+---
+
+
+## 🚀 **New Features**
+
+### 🔄 **Enable/Disable Sync**
+We’ve introduced the ability to enable or disable a sync. When a sync is disabled, it won’t execute according to its schedule, allowing you to effectively pause it without the need to delete it. This feature provides greater control and flexibility in managing your sync operations.
+
+### 🧠 **Databricks AI Model Connector**
```
Comment (afthabvp) [inline release-notes/August_2024.mdx:13]: Source: Databricks AI Model Connector

File: release-notes/August_2024.mdx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+## 🔧 **Improvements**
+
+### **Catalog Creation via API**
+Enhanced API support for catalog creation, streamlining workflows for developers and system integrators by automating catalog management.
+
+### **Catalog Update API**
+The catalog update API has been improved, enabling more seamless updates to existing catalogs, ensuring that your data is always current.
+
+### **Databricks Model Connection Specification**
```
Comment (afthabvp) [inline release-notes/August_2024.mdx:41]: remove this

File: release-notes/August_2024.mdx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+### **Logger Issue in Sync**
+Addressed a logging issue within sync operations, ensuring that logs are accurate and provide valuable insights.
+
+### **Main Layout Protected**
+Wrapped the main layout with a protector, enhancing security and stability across the platform.
+
+### **User Email Verification**
+Implemented email verification during signup using Devise, increasing account security and ensuring that only verified users have access.
+
+### **Databricks Datawarehouse Name Change**
```
Comment (afthabvp) [inline release-notes/August_2024.mdx:89]: its also part of Connector Name Update

File: release-notes/August_2024.mdx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+Renamed the Databricks connection to "Databricks Datawarehouse" for improved clarity and better alignment with user expectations.
+
+### **Integration Gem Update**
```
Comment (afthabvp) [inline release-notes/August_2024.mdx:92]: provide only the most recent gem upgrade. remove others

File: release-notes/August_2024.mdx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+---
+
+## 🔧 **Improvements**
+
+### **Catalog Creation via API**
+Enhanced API support for catalog creation, streamlining workflows for developers and system integrators by automating catalog management.
+
+### **Catalog Update API**
+The catalog update API has been improved, enabling more seamless updates to existing catalogs, ensuring that your data is always current.
```
Comment (subintp) [inline release-notes/August_2024.mdx:39]: ??

File: release-notes/August_2024.mdx [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+### 🗄️ **Oracle DB as a Source Connector**
+[Oracle DB](https://docs.squared.ai/guides/data-integration/sources/oracle) has also been added as a source connector. Users can pull data from Oracle databases by navigating to **Add Source**, selecting **Oracle**, and entering the database details.
+
+---
+
+## 🔧 **Improvements**
+
+### **Catalog Creation via API**
+Enhanced API support for catalog creation, streamlining workflows for developers and system integrators by automating catalog management.
```
Comment (subintp) [inline release-notes/August_2024.mdx:36]: ???

**Review by sumitd94 — APPROVED** [review]

**Review by afthabvp — APPROVED** [review]

**Pull request merged**